KIClubinfo/upont-v4
# Post resource

## Features

* Possibility to create a post for a course on the course details page
* Adding a file to a course post will create a resource wich name is the name of the file (may be changed later)
* The number of resources by post is limited to one. I'm open to discussion if you think this has to change.

## To test

*  Assert that the "news" posts work like before (create/edit/delete)
*  Check that all the operations on "course" posts work (create/edit/delete), and check if resources are created and deleted correctly.
